portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,29 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> messages;
+
+  public void initializeMessages() {
+    messages = new ArrayList<String>();
+    messages.add("Welcome to my page!");
+    messages.add("Porfolio time!");
+    messages.add("This is my portfolio");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    initializeMessages();
```

Comment:
Question: Do we need to initializeMessages() every time doGet method is called?

Reply:
No - just fixed this in recent commit